### PR TITLE
Adding new blog categories

### PR DESCRIPTION
**Pull request by SumeyraBulut**

### CODE REVIEW

File: src/routes/blog/category/company/+page.markdoc
Comment (LauraDuRy) [inline src/routes/blog/category/company/+page.markdoc:1]: this is more about company updates like funding rounds, big changes, news type of content. What you explained is more part of culture

File: src/routes/blog/category/community/+page.markdoc
Comment (LauraDuRy) [inline src/routes/blog/category/community/+page.markdoc:1]: Let's remove community, I see we also have contributors, open source, and case studies, comunity is overkill

File: src/routes/blog/category/devrel/+page.markdoc
Comment (LauraDuRy) [inline src/routes/blog/category/devrel/+page.markdoc:1]: Remove `dive into` from all of them, sounds a bit off.

Make all of them less wordy and more to the point. So example: `Read about developer relations and get tips and insights.`

File: src/routes/blog/category/integrations/+page.markdoc
Comment (LauraDuRy) [inline src/routes/blog/category/integrations/+page.markdoc:1]: Again, too wordy and a lot of fluf. Make it more to the point: Learn more about integrations and Appwrite

File: src/routes/blog/category/security/+page.markdoc
Comment (LauraDuRy) [inline src/routes/blog/category/security/+page.markdoc:1]: Same

File: src/routes/blog/category/tutorial/+page.markdoc
Comment (LauraDuRy) [inline src/routes/blog/category/tutorial/+page.markdoc:1]: Learn how to implement Appwrite into your projects.

**Review by LauraDuRy — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]
